sdk/migrationdiscoverysap/Azure.ResourceManager.MigrationDiscoverySap/tests/Tests/MigrationSapDiscoveryTests.cs

```diff
@@ -146,13 +145,16 @@ public async Task PostImportEntities(ArmClient client, SapDiscoverySiteResource
         var operationStatusObj = JObject.Parse(opStatus.Content.ToString());
         var inputExcelSasUri = operationStatusObj?["properties"]?["discoveryExcelSasUri"].ToString();
```

Comment:
should this be included into the recording run as well? while playback mode, this is not needed.

Reply:
Done